### DISCUSSION TITLE
'NoneType' Object is not subscriptable

### DISCUSSION THREAD

**Discussion by matixmatisse** [FabCx/TradingView-Webhook-Bot#56] [General]
I've been troubleshooting for hours.
Have my webhook URL input:
https://discordapp.com/api/webhooks/847676475314536449/FJpcMXa153ZfMIZNfXedh-7PMrlp8jPD6Hk0Ex3J9I0ZuvrXJsHsxIUe0ptCHxfgFo8F'

Have my secret key matching.
Have this placed in my Discord field:
"847676475314536449/FJpcMXa153ZfMIZNfXedh-7PMrlp8jPD6Hk0Ex3J9I0ZuvrXJsHsxIUe0ptCHxfgFo8F"

What am I doing wrong. Using insomnia and it just gives me an error any time I try to fire a post.

**Comment by fabston** [comment]
Hi,

The error means your json string is most likely wrongly formatted. Please post your syntax here and I can take a look.

**Reply by matixmatisse** [reply]
Hi,

I actually got Insomnia to fire a perfect message to Discord, but now when I input the same syntax into TradingView message box I get this error: 
https://gyazo.com/6860aa91faad6d350846097447849dce

Message syntax here:
{
 "key": "somelongstring123",
 "discord": "EXCLUDING MY INFORMATION, IT"S CORRECT",
 "msg": "Long *#{{ticker}}* at `{{close}}`"
}

**Reply by matixmatisse** [reply]
Oh wow I'm an idiot.. I had the webhook URL in the alert name. All is well now!

**Reply by fabston** [reply]
Ha, ok cool!

**Reply by matixmatisse** [reply]
Is there a way to include a snapshot of the chart? That way people can see the actual graph alongside the alert?
@fabston

**Reply by fabston** [reply]
Yes that’s possible but it is not open sourced, you’d have to use the service I am offering: https://webhookalerts.com

**Reply by matixmatisse** [reply]
Ah, ok.. was hoping you'd be able to help me out. Will try to figure it out on my own! Thanks!

**Reply by fabston** [reply]
Sure, good luck.

**Reply by matixmatisse** [reply]
For anyone looking to do the same - it IS open source:
https://github.com/itsaLca/Advanced-Tradingview-Webhook

**Reply by fabston** [reply]
All I said is, I did not open source it.
